### PR TITLE
Add Ambassador Edge Stack to Rancher's partner charts

### PR DESCRIPTION
**Pull request by cakuros**
Hi!  This is a request to add Ambassador Edge Stack as a partner chart in Rancher's managed apps.  I'm pretty sure I followed the readme instructions, but let me know if I missed anything, or if you have any questions.  I did test the chart itself and multiple variations of the questions on Rancher Cluster Explorer v2.5.7.  Thanks!

### CODE REVIEW

File: packages/ambassador/ambassador.patch [hunk 1]
```diff
@@ -0,0 +1,10 @@
+diff -x '*.tgz' -x '*.lock' -uNr packages/ambassador/charts-original/Chart.yaml packages/ambassador/charts/Chart.yaml
+--- packages/ambassador/charts-original/Chart.yaml
++++ packages/ambassador/charts/Chart.yaml
+@@ -1,3 +1,6 @@
++annotations:
++  catalog.cattle.io/certified: partner
++  catalog.cattle.io/release-name: ambassador
```
Comment (PennyScissors) [inline packages/ambassador/ambassador.patch:7]: Could you please add this additional annotation for the name that will be displayed in the UI?
`catalog.cattle.io/display-name: Cakuros Ambassador`
I used Cakuros Ambassador as an example, but you can change it to whatever you think is best.

**Review by PennyScissors — COMMENTED** [review]
Looks good overall, just one small addition and it should be ready to go. Good job! 😄

File: packages/ambassador/ambassador.patch (hunk 1 above)
Comment (cakuros) [reply packages/ambassador/ambassador.patch:7]: Done!

**Comment by cakuros** [comment]
Added the display name annotation, and updated to the most recent chart version.

**Review by PennyScissors — APPROVED** [review]

**Pull request merged**